### DISCUSSION TITLE
z-ui logs no rotate

### DISCUSSION THREAD

**Discussion by rcrocus** [zwave-js/zwave-js-ui#3609] [Q&A] [answered]
Hello Everyone,

I use Zwave-js UI on raspberry (zwave-js-ui: 9.8.2.c09b638 & zwave-js: 12.4.3).
The logrotate is OK for zwavejs.log but no for z-ui logs.
In log directory, I have zwavejs_logrotate.json but I haven't z-ui_logrotate.json (instead I have this file : **zui-logs.audit.json.**)
The z-ui logs are compressed every day (*.log.gz) but all daily log are present.

Do you know if I can add a z-ui_logrotate.json ? If yes, do you have an example ?

Thanks
Romain

**Comment by robertsLando** [comment]
> instead I have this file : zui-logs.audit.json.)

That's the same as `zwavejs_logrotate.json` if you check the content. Anyway I will check this and come back to you in case I'm able to reproduce this issue

**Comment by rcrocus** [comment]
I have only one difference. In **zwavejs_logrotate.json**, I have this line (but no in **zui-logs.audit.json**) :    
"hashType": "sha256"

**Comment by robertsLando** [comment]
Speaking with others seems this is an issue related to the package that handles the rotation of files. I will submit a workaround to fix this

**Comment by rcrocus** [comment]
Thanks for your help.
When the fix is commited, tell me and I'll try it !

**Comment by rcrocus** [comment]
Hello, do you know when you can submit this workaround ?
Thanks.

**Reply by robertsLando** [reply, marked as answer]
Already submitted it's available starting from 9.9.1

**Reply by rcrocus** [reply]
Thanks for answer.

I've tested the new version and it works well !
Thanks
